bin/changelog.js

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env node
+'use strict';
+
+const requestPromise = require('request-promise');
+const chalk = require( 'chalk' );
+const octokit = require( '@octokit/rest' )();
+const promptly = require('promptly');
+
+const headers = {
+	'Content-Type': 'application/json;charset=UTF-8',
+	'Authorization': `token ${process.env.GH_API_TOKEN}`,
+	'Accept': 'application/vnd.github.inertia-preview+json',
+	'User-Agent': 'request'
+};
+
+const columnIds = [];
+
+const getPullRequestType = labels => {
+	const typeLabel = labels.find( label => label.name.includes( '[Type]' ) );
+	if ( ! typeLabel ) {
+		return 'Dev';
+	}
+	return typeLabel.name.replace( '[Type] ', '' );
+};
+
+const getLabels = labels => {
+	return labels
+		.filter( label => ! /\[.*\]/.test( label.name ) )
+		.map( label => label.name )
+		.join( ', ' );
+
+};
+const isCollaborator = async ( username ) => {
+	return requestPromise( {
+		url: `https://api.github.com/orgs/woocommerce/members/${ username }`,
+		headers,
+		resolveWithFullResponse: true
+	} ).then( response => {
+		return response.statusCode === 204;
+	} )
+		.catch( err => {
+			if ( err.statusCode !== 404 ) {
+				console.log( '🤯' );
+				console.log( err.message );
+			}
+		});
+}
+
+const writeEntry = async ( content_url ) => {
+	const options = {
+		url: content_url,
+		headers,
+		json: true
+	}
+	return requestPromise( options )
+		.then( async data => {
+			if ( data.pull_request ) {
+				const collaborator = await isCollaborator( data.user.login );
+				const type = getPullRequestType( data.labels );
+				const labels = getLabels( data.labels );
+				const labelTag = labels.length ? `(${ labels })` : '';
+				const authorTag = !! collaborator ? '' : `👏 @${ data.user.login }`;
```

Comment:
If I'm reading this correctly, `collaborator` will always be a boolean or `undefined`, right? Do you think we could get rid of the double negation or am I missing something?
```suggestion
				const authorTag = collaborator ? '' : `👏 @${ data.user.login }`;
```

Reply:
👍